templates/bun/server.js

```diff
@@ -0,0 +1,39 @@
+import * as Bun from "bun";
+import * as fs from "node:fs";
+
+import {
+  broadcastDevReady,
+  createRequestHandler,
+} from "@remix-run/server-runtime";
+
+const BUILD_PATH = "./build/index.js";
+const STATIC_PATH = "./public";
+
+let build = await import(BUILD_PATH);
+if (build.dev) {
+  broadcastDevReady(build);
+}
+
+export default {
+  async fetch(request) {
+    const url = new URL(request.url);
+
+    try {
+      const filePath = STATIC_PATH + url.pathname;
+      if (fs.statSync(filePath).isFile()) {
```

Comment:
Instead of this, you can run `Bun.file(filePath)` and then `await file.exists()` to know if the path points to a real file.